### DISCUSSION TITLE
`Isomorphic Resolver Imports` looks gone in IDE, but docs still has.

### DISCUSSION THREAD

**Discussion by dhythm** [blitz-js/blitz#3107] [Q&A] [answered]
Thanks for all contributors and participants.
I have a question.

I try to upgrade blitz.js to v0.44.4. Then, I noticed the following issue.

The docs still has the description of `isomorphic resolver imports`.
https://blitzjs.com/docs/blitz-config#isomorphic-resolver-imports

However, the type in `node_modules/next/dist/server/config-shared.d.ts` doesn't have it.
![image](https://user-images.githubusercontent.com/10891318/148014240-a72cdf84-e2b1-4f5b-b876-40142ad89942.png)

Has the `experimentals.isomorphicResolverImports` already been outdated?
Or is it moved to somewhere?

Thanks for your kind support 🙏

**Comment by beerose** [comment, marked as answer]
Hey! The `isomorphicResolverImports` is gone and should be removed from docs. I'll convert it to an issue in the docs repo.

**Reply by beerose** [reply]
Actually, I'll create a new issue in docs repo, as I can't transfer the discussion there directly 😅

**Reply by dhythm** [reply]
Sure. Thanks a lot! 🤓
